hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/tracing/Span.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.tracing;
+
+import io.opentracing.SpanContext;
+import java.io.Closeable;
+
+public class Span implements Closeable {
```

Comment:
Could you please add javadoc that describes the usage of this class?

Reply:
Thanks Adam for the comment.

Yeah we would definitely do that if we are going to keep the shim. But now we are going to remove it, so the class won't exist anymore.